### PR TITLE
Add link generator-jhipster-module to the page creating a module

### PR DESCRIPTION
**Pull request by pascalgrimaud**
As suggested by @geraldhumphries in this https://github.com/jhipster/jhipster.github.io/pull/159
It will help users who decide to create a module to start faster

### CODE REVIEW

File: modules/creating_a_module.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ The [JHipster Fortune module](https://github.com/jdubois/generator-jhipster-fort
 
 It is our sample module that showcases how you can JHipster's variables and functions in order to create your own generator.
 
+Or, you can use the [JHipster module](https://github.com/pascalgrimaud/generator-jhipster-module) to help you to initialize your module.
```
Comment (jdubois) [inline modules/creating_a_module.md]: "JHipster module" doesn't look very clear to me

Comment (pascalgrimaud) [inline modules/creating_a_module.md]: What do you suggest ? Generator JHipster module ?
Or maybe do you want I rename my module (repository, npm, etc) ?

Comment (pascalgrimaud) [inline modules/creating_a_module.md]: And if you prefer this module to be official, in JHipster organization, don't hesitate! Just ask. I can donate this project (I think it's possible, right ?)

Comment (jdubois) [inline modules/creating_a_module.md]: Yes, maybe like "JHipster module generator"
You can "donate" it to the project easily (it's just a repo transfer), and indeed this should be official, but I haven't had a look at it yet

Comment (pascalgrimaud) [inline modules/creating_a_module.md]: So, it's ok, don't merge it yet.
As soon as you valid the module, I will donate, and update this PR.

**Comment by deepu105** [comment]
I also think that this should be official, I dont like the yeoman generator-generator very much, with this we can make sure people are following same pattern etc.

I would like to make some improvements in it as well once I find time

**Comment by pascalgrimaud** [comment]
Yes, I code this for the fun, during happy holidays, with reference to Lord Of The Ring :ring:
If it become official, it should be more serious...

**Comment by cbornet** [comment]
Well, a genrerator that generates other generators is more like "The Matrix" to me :smile:

**Comment by jdubois** [comment]
So, should I merge this PR or not?
@pascalgrimaud if you want to transfer it to the JHipster organization it's of course most welcome, and of course I will update the rights so you can continue to work on it as of today

**Comment by pascalgrimaud** [comment]
@jdubois : don't merge!
I will donate this to the JHipster organization, then I will update the link in this PR

**Comment by pascalgrimaud** [comment]
I transfert it to you, instead of JHipster organization => https://github.com/jdubois/generator-jhipster-module. U'll have to transfert it to JHipster organization
(I have only web interface during the day...)

**Comment by jdubois** [comment]
Oh, OK!

**Comment by pascalgrimaud** [comment]
- PR : link updated, so you can merge
- [npmjs](https://www.npmjs.com/package/generator-jhipster-module/) : you are a collaborator to the project, you should be able to npm publish
- the package.json need to be updated (I won't be able to do this, until tonight)

**Comment by pascalgrimaud** [comment]
One last thing : maybe @deepu105 you want to be main collaborator like me with same rights ?

**Comment by deepu105** [comment]
Yup sure i will add some template for hooks and stuff

**Pull request merged**